docs/development/vsphere-dev.md

```diff
@@ -56,6 +56,17 @@ ns1.ns.dns.skydns.local. 160	IN	A	192.168.0.1
 ### Add DNS server information when create cluster
 Add ```--dns=private --vsphere-coredns-server=http://[DNS server's IP]:2379``` into the ```kops create cluster``` command line.
 
+### Use CoreDNS supported DNS Controller
+Information about DNS Controller can be found [here](https://github.com/kubernetes/kops/blob/master/dns-controller/README.md)
+Currently the DNS Controller is an add-on container and the image is from kope/dns-controller.
+Before the vSphere support is officially merged into upstream, we need to set up CoreDNS supported DNS controller manually.
+```bash
+DOCKER_REGISTRY=[your docker hub repo] make dns-controller-push
```

Comment:
No this can be one.
Even without modifying the DOCKER_REGISTRY inside Makefile, you can run command like:
"DOCKER_REGISTRY=[your docker hub repo] make dns-controller-push"
or
"DOCKER_REGISTRY=[your docker hub repo] make push-vsphere"
Whatever you set here will override the one inside Makefile's default value.